pnpm-lock.yaml

```diff
@@ -10785,12 +11039,12 @@ snapshots:
   codemirror@6.0.1:
     dependencies:
       '@codemirror/autocomplete': 6.18.6
-      '@codemirror/commands': 6.8.0
+      '@codemirror/commands': 6.8.1
       '@codemirror/language': 6.10.8
       '@codemirror/lint': 6.4.2
       '@codemirror/search': 6.5.3
       '@codemirror/state': 6.5.2
-      '@codemirror/view': 6.36.4
+      '@codemirror/view': 6.36.5
```

Comment:
For some reason, dependabot updates both the affected dep (Vite) and also its dependencies (rollup, which is understandable I guess). But then it also updates random unnecessary deps like `@codemirror` packages here. It might be actually a pnpm issue (https://github.com/pnpm/pnpm/issues/5552#issuecomment-2428653185).

In any case, I reverted the extra updates, leaving only the Vite update.